### PR TITLE
added changelog

### PR DESCRIPTION
**Pull request by bartes**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Change Log
+
+## v3.0.0 (draft)
```
Comment (nijikon) [inline CHANGELOG.md]: `draft` -> `unreleased`

**Comment by nijikon** [comment]
Also let's drop the `v` so `v3.0.0` -> `3.0.0`

**Comment by bartes** [comment]
this how they are tagged and displayed in releases - I wanted to keep them in the same format

**Comment by bartes** [comment]
draft -> unreleased .. done

**Comment by bartes** [comment]
dropped v prefix as well

**Pull request merged**